src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/WorkflowSwitch.html

```diff
@@ -0,0 +1,6 @@
+<div class="workflow-switch" data-ng-class="{ 'has-switch': availableStates }">
+    <div class="workflow-switch-transcluded" data-ng-transclude=""></div>
+    <div class="workflow-switch-container" data-ng-if="availableStates">
```

Comment:
In JavaScript, `[]` is truthy. A simple way to get the behavior you want is `availableStates.length` because `0` is falsy and angular expressions are forgiving (`undefined.length` will be falsy instead of throwing an exception).